crates/vhost/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 
 ### Deprecated
 
+## [0.6.1]
+- [[#182]](https://github.com/rust-vmm/vhost/pull/182) Fix set_vring_addr issues
```

Comment:
Please follow the style adding a "Fixed" section

Reply:
Fixed.